testing/travis-fork-check.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import logging
+import os
+import shlex
+import subprocess
+import sys
+
+def check_travis_repo_slug():
+    if 'encrypted_6a9d32f3e0bd_key' not in os.environ:
```

Comment:
Do they follow some sort of pattern? If we change keys the ID could change and have an false positive. Do they follow `encrypted_xxx_key`?

`any(key.startswith('encrypted') and key.endswith('key') for key in os.environ)` or something like it.